tests/integration_test.go

```diff
@@ -83,6 +82,15 @@ func mineMinXMRMakerBalance(t *testing.T, minBalance common.MoneroAmount) {
 	}
 }
 
+func (s *IntegrationTestSuite) newSwapdWSClient(ctx context.Context, endpoint string) wsclient.WsClient {
+	wsc, err := wsclient.NewWsClient(ctx, endpoint)
+	require.NoError(s.T(), err)
+	s.T().Cleanup(func() {
+		wsc.Close()
+	})
+	return wsc
+}
```

Comment:
We were not closing the websocket connections when the tests ended, so I added this and converted all the tests to use it.